about.html

```diff
@@ -0,0 +1,182 @@
+          </p>
+          <div class="bg-white">
+          <h3 class="logo-design">
+            Design of Erriecha Festival 2023 Logo
+          </h3>
+          <p class="logo-designer">
+            The logo of Erriecha Was designed by Osman Wako Full Stack Software Developer.
+          </p>
+          <div class="festival-logo">
+            <img src="./assets/images/titleicon.png" alt="Irreechaa Logo" class="festival-images">
+          </div>
+        <hr>
+        </div>
+        </div>
+      </section>
+      <section>
+        <div class="start-align end-align">
+          <h1 class="seepastyear"><br>See The Past Erriecha Festival</h1>
+        <p class="par-history seepastyear">
+          Take a look at the recent two Erriecha Festivals which took place in Hora Finfinne and Hora Har-sadii.
+        </p>
+        <div class="grid-photo">
+          <div class="grid-item grid-album-1">
+          <img src="./assets/images/festival/fest1.jpg" class="fest-img" >
+            <p class="overlay-text">2022<br>Erriecha Annual Festival</p>
+          </div>
+          <div class="grid-item grid-album-1">
+            <img src="./assets/images/festival/fest3.jpg" class="fest-img">
+            <p class="overlay-text">2020<br>Erriecha Annual Festival</p>
+          </div>
+          <div class="grid-item grid-album-1">
+            <img src="./assets/images/festival/fest2.png" class="fest-img">
+            <p class="overlay-text">2019<br>Erriecha Annual Festival</p>
+          </div>
+        </div>
+        </div>
+      </section>
+    </main>
+    <footer class="">
+      <section class="footer-partner bg-main">
+        <div class="partner-head start-align end-align">
+          <h2 class="partner-text txt-second">
+            Partner
+           </h2>
+        </div>
+         <div class="partner-log start-align end-align txt-main">
+             <img class="partner-image" src="./assets/images/partner/abyssiniabank.png">
+             <img class="partner-image" src="./assets/images/partner/awashbank.svg">
+             <img class="partner-image" src="./assets/images/partner/nibbank.png">
+             <img class="partner-image" src="./assets/images/partner/oromiabank.png">
+             <img class="partner-image" src="./assets/images/partner/microverse.png">
+         </div>
+      </section>
+      <section class="footer-logo bg-second ">
+         <div class="footerlogo-icon start-align">
+            <img src="./assets/images/titleicon.png" alt="Festival logo" class="event-logo">
+          </div>
+          <div class="footerevent-message end-align txt-main">
+             <p>Irreechaa  Love,Peace and Unity</p>
+             <p>
+              <br>
+              CELEBRATING IRREECHAA FESTIVAL 2023
+             </p>
+          </div>
+         </div>
+      </section>
+    </footer>
```

Comment:
- [ ] Kindly remove the partner section on the about page for desktop version and change the background color of the footer to dark as we have it on the guiding template. 

### Your Design
![your-partner-about](https://user-images.githubusercontent.com/26597432/226097201-72435d17-4e04-409a-b8af-4f12659c20cd.png)

### Guiding Template
![temp-about-footer](https://user-images.githubusercontent.com/26597432/226097216-d762421e-da1b-4aca-9c3f-9703cb957a26.png)